CanDIG/katsu
# Sonchau/fix docs commit sha

- Fixed: incorrect SHA reference for the schema.
- Improved: instead of triggering on PR review, it now triggers on schema and model changes. This helps to reduce unnecessary auto-update

How to test:
- Make any changes to a schema. For example: 
```
class DonorWithClinicalDataSchema(ModelSchema):
    primary_diagnoses_new_name: List[NestedPrimaryDiagnosisSchema] = Field(
        None, alias="primarydiagnosis_set"
    )
```
- Push
- The schemas should update with the new SHA, copy that link and open `schema.json` to verify `primary_diagnoses_new_name` added
